tests/helpers/mod.rs

```diff
@@ -21,27 +23,31 @@ impl TestApp {
     }
 
     fn copy_demo_files(&self) {
-        std::fs::write(
-            self.tmp_path.join("demo.py"),
-            include_str!("../../demo/demo.py"),
-        )
-        .expect("");
-        std::fs::write(
-            self.tmp_path.join("test_demo.py"),
-            include_str!("../../demo/test_demo.py"),
-        )
-        .expect("");
-        std::fs::write(
-            self.tmp_path.join("setup.py"),
-            include_str!("../../demo/setup.py"),
-        )
-        .expect("");
+        for result in Walk::new("demo") {
+            match result {
+                Ok(entry) => {
+                    if let Some(file_type) = entry.file_type() {
+                        if file_type.is_file() {
+                            let src = entry.path();
+                            let name = entry.file_name();
+                            let dest = self.tmp_path.join(name);
+                            std::fs::copy(src, dest).expect("");
+                        }
+                    }
+                }
+                Err(e) => panic!(e),
```

Comment:
If you're going to panic on err, maybe instead of a match which adds 2 levels of indetation, you can `let entry = result.unwrap()` on line 2?